src/mjolnir/linkclassification.cc

```diff
@@ -9,6 +9,8 @@
 #include "baldr/graphid.h"
```

Comment:
at a high level its looks pretty much like a rewrite of the reclassification (which isnt a bad thing) but i was wondering if you might be able to add some comments to this file to draw our attention to the flow of the algorithm. basically explain how this new version works?

Reply:
sure!

Reply:
@kevinkreiser added some more detailed comments. feel free to ask if you need some more explanations)